example/ios/Podfile

```diff
@@ -54,3 +74,13 @@ target 'RNPermissionsExample' do
 
   use_native_modules!
 end
+
+# post_install do |installer|
```

Comment:
Id this needed in any way or can be removed?

Reply:
oh, sorry
i have forgot to remove dead code
i will do new PR soon